From dc4237cdec4e09709b13c15ba09f2f8715b3c173 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C5=A0t=C4=9Bp=C3=A1n=20N=C4=9Bmec?= <stepnem@gmail.com>
Date: Mon, 7 Mar 2022 23:03:55 +0100
Subject: [PATCH attr] Fix stale attr(5) references
Origin: vendor
Forwarded: yes
Applied-Upstream: 2.5.1+, commit:dc4237cdec4e09709b13c15ba09f2f8715b3c173
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The man page has been named xattr(7) since 2015, cf. the following
man-pages commit:

 commit 43834827b827
 Author: Michael Kerrisk <mtk.manpages@gmail.com>
 Date:   Wed Apr 22 14:29:42 2015 +0200

     xattr.7: Rename from attr.7

Signed-off-by: Štěpán Němec <stepnem@gmail.com>
---
 README              | 2 +-
 man/man1/attr.1     | 2 +-
 man/man1/getfattr.1 | 4 ++--
 man/man1/setfattr.1 | 2 +-
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/README b/README
index 006c344..937e394 100644
--- a/README
+++ b/README
@@ -6,7 +6,7 @@ Package home: http://savannah.nongnu.org/projects/attr
 See the file doc/INSTALL for build, installation and post-
 install configuration steps.
 
-Refer to the attr(5) manual page for general extended attribute
+Refer to the xattr(7) manual page for general extended attribute
 (EA) information and references to other EA manual pages.
 
 For more information on the build process, please refer to
diff --git a/man/man1/attr.1 b/man/man1/attr.1
index 84e80de..ebbd2ef 100644
--- a/man/man1/attr.1
+++ b/man/man1/attr.1
@@ -210,5 +210,5 @@ or
 .BR attr_set (3),
 .BR attr_multi (3),
 .BR attr_remove (3),
-.BR attr (5),
+.BR xattr (7),
 .BR xfsdump (8)
diff --git a/man/man1/getfattr.1 b/man/man1/getfattr.1
index 95da3fb..df8bb8a 100644
--- a/man/man1/getfattr.1
+++ b/man/man1/getfattr.1
@@ -90,7 +90,7 @@ The default value for
 is "^user\\\\.", 
 which includes all the attributes in the user namespace. Specify "\-" for
 including all attributes.  Refer to
-.BR attr (5)
+.BR xattr (7)
 for a more detailed discussion of namespaces.
 .TP
 .B \-\-absolute-names
@@ -139,4 +139,4 @@ or
 .RI < acl-devel@nongnu.org >.
 .SH "SEE ALSO"
 .BR setfattr (1),
-.BR attr (5)
+.BR xattr (7)
diff --git a/man/man1/setfattr.1 b/man/man1/setfattr.1
index 50dd58f..e011ca7 100644
--- a/man/man1/setfattr.1
+++ b/man/man1/setfattr.1
@@ -118,4 +118,4 @@ or
 .RI < acl-devel@nongnu.org >.
 .SH "SEE ALSO"
 .BR getfattr (1),
-.BR attr (5)
+.BR xattr (7)
-- 
2.40.0

